data-model-translator-wrapper/src/main/java/ai/aitia/arrowhead/dmtpw/api/http/DataModelTranslationAPI.java

```diff
@@ -0,0 +1,112 @@
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import ai.aitia.arrowhead.Constants;
+import ai.aitia.arrowhead.dmtpw.DataModelTranslatorWrapperConstants;
+import ai.aitia.arrowhead.dmtpw.service.DataModelTranslationService;
+import eu.arrowhead.dto.DataModelTranslationInitRequestDTO;
+import eu.arrowhead.dto.DataModelTranslationResultResponseDTO;
+import eu.arrowhead.dto.ErrorMessageDTO;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.security.SecurityRequirement;
+
+@RestController
+@RequestMapping(DataModelTranslatorWrapperConstants.HTTP_API_DATA_MODEL_TRANSLATION_BASE_PATH)
+@SecurityRequirement(name = Constants.SECURITY_REQ_AUTHORIZATION)
+public class DataModelTranslationAPI {
+
+	//=================================================================================================
+	// members
+
+	@Autowired
+	private DataModelTranslationService translationService;
+
+	private final Logger logger = LogManager.getLogger(this.getClass());
+
+	//=================================================================================================
+	// methods
+
+	//-------------------------------------------------------------------------------------------------
+	@Operation(summary = "Initiates the data model translation job and returns its ID")
+	@ApiResponses(value = {
+			@ApiResponse(responseCode = Constants.HTTP_STATUS_OK, description = Constants.SWAGGER_HTTP_200_MESSAGE, content = {
+					@Content(mediaType = MediaType.TEXT_PLAIN_VALUE, schema = @Schema(implementation = UUID.class)) }),
+			@ApiResponse(responseCode = Constants.HTTP_STATUS_BAD_REQUEST, description = Constants.SWAGGER_HTTP_400_MESSAGE, content = {
+					@Content(mediaType = MediaType.APPLICATION_JSON_VALUE, schema = @Schema(implementation = ErrorMessageDTO.class)) }),
+			@ApiResponse(responseCode = Constants.HTTP_STATUS_INTERNAL_SERVER_ERROR, description = Constants.SWAGGER_HTTP_500_MESSAGE, content = {
+					@Content(mediaType = MediaType.APPLICATION_JSON_VALUE, schema = @Schema(implementation = ErrorMessageDTO.class)) }),
+			@ApiResponse(responseCode = Constants.HTTP_STATUS_SERVICE_UNAVAILABLE, description = Constants.SWAGGER_HTTP_503_MESSAGE, content = {
```

Comment:
I think this is not necessary in your case because your code not call an other service.